pkg/terraform/stages/ovirt/stages.go

```diff
@@ -0,0 +1,16 @@
+package ovirt
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/stages"
+	ovirttypes "github.com/openshift/installer/pkg/types/ovirt"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in oVirt.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage(ovirttypes.Name, "image", stages.WithNormalDestroy()),
+	stages.NewStage(ovirttypes.Name, "cluster"),
+	stages.NewStage(ovirttypes.Name, "bootstrap", stages.WithNormalDestroy()),
+}
+
+// Note that the bootstrap IP and control-plane IP extraction is not supported on oVirt
```

Comment:
We support extracting the bootstrap IP with: https://github.com/openshift/installer/blob/master/pkg/terraform/gather/ovirt/ip.go#L82-L114

We didn't implement control-plane IPs extraction and I think we just forgot about it since we can, @eslutsky is there a reason for not adding control plane IPs?

Reply:
the original PR was for extracting IP information for bootstrap VM only, so i guess we missed it,
It should be an easy fix, I'll open PR for this.

Reply:
OK please link it here, I think we can still call the function on this PR just to not be blocked on this

Reply:
https://github.com/openshift/installer/pull/5039